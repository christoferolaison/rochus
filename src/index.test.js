import splitStyleProps from '.'

test('should return an empty props object and a sx object containing display key', () => {
  expect(splitStyleProps({ display: 'flex' })).toMatchSnapshot()
})

test('should return an props object containing onClick and a sx object containing alignItems key', () => {
  expect(splitStyleProps({ alignItems: 'center', onClick: () => {} })).toMatchSnapshot()
})
