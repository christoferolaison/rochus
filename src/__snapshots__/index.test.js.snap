// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return an empty props object and a sx object containing display key 1`] = `
Object {
  "props": Object {},
  "sx": Object {
    "alignItems": undefined,
    "backgroundColor": undefined,
    "color": undefined,
    "display": "flex",
    "flex": undefined,
  },
}
`;

exports[`should return an props object containing onClick and a sx object containing alignItems key 1`] = `
Object {
  "props": Object {
    "onClick": [Function],
  },
  "sx": Object {
    "alignItems": "center",
    "backgroundColor": undefined,
    "color": undefined,
    "display": undefined,
    "flex": undefined,
  },
}
`;
